jdi-light-html-tests/src/main/java/io/github/com/pages/HtmlElementsPage.java

```diff
@@ -97,7 +105,8 @@
     public static MultiSelector ages;
     //@UI("#ages") public static MultiSelector ages;
     public static MultiSelector multiDropdown;
-    // @UI("#multi-dropdown") public static MultiSelector multiDropdown;
+    // @UI("#multi-dropdown") public static
+    // public static multiDropdown;
```

Comment:
Please return to previous variant on one line

Reply:
recovered to:
"// @UI("#multi-dropdown") public static multiDropdown;"